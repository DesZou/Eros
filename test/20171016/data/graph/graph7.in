50 600 71
20 9 342
25 27 712
13 1 23
11 48 654
37 34 707
4 19 944
6 11 943
8 3 2
29 48 119
2 10 426
16 14 604
45 36 348
30 35 188
1 2 826
19 13 909
34 9 955
2 13 523
31 45 10
36 48 305
11 45 405
22 13 983
21 29 913
50 33 345
49 32 256
49 46 704
36 30 343
36 38 309
19 35 742
42 7 952
19 5 922
13 3 656
17 1 46
27 40 952
32 31 945
4 24 939
33 16 546
9 47 703
44 24 985
17 31 822
12 14 781
36 11 940
49 1 28
25 37 431
35 17 571
34 40 935
31 40 492
25 44 4
1 9 805
10 1 718
23 18 161
40 25 219
34 17 45
47 33 956
30 11 233
40 49 41
22 42 366
35 22 709
44 20 849
14 50 435
10 17 593
47 31 335
21 9 770
42 33 456
45 43 742
14 5 423
8 48 680
1 47 665
11 43 512
10 5 497
16 24 538
45 21 34
5 7 270
15 16 65
41 31 958
43 31 419
27 23 665
14 22 306
18 29 415
1 22 820
16 20 913
25 35 135
34 14 162
5 39 212
36 34 993
6 15 416
31 35 346
26 2 718
5 43 696
13 25 594
4 23 989
18 38 76
40 21 174
47 26 384
41 49 402
1 24 736
24 25 41
2 6 136
41 34 673
6 47 963
26 29 308
6 29 339
15 43 899
12 30 802
43 39 319
17 4 398
46 7 621
24 3 339
29 46 575
32 16 733
10 37 431
29 4 107
33 44 136
50 45 198
44 28 888
50 37 607
18 4 983
15 7 163
21 24 796
26 49 790
46 40 166
7 12 470
20 6 114
30 50 196
18 44 5
6 9 669
42 37 202
11 42 339
37 13 10
13 35 201
50 47 329
12 41 184
22 34 877
12 11 260
23 15 580
23 41 332
30 25 191
12 23 202
23 11 376
31 48 869
1 28 365
33 5 604
39 10 23
49 28 961
48 25 365
14 1 48
47 40 230
46 21 897
49 25 343
2 36 698
42 39 994
43 2 667
7 24 565
2 5 370
21 31 925
36 24 712
49 23 497
19 29 821
49 24 116
36 3 524
25 32 47
32 19 21
44 38 34
26 21 494
48 50 545
22 31 339
12 3 956
45 35 295
34 13 390
30 15 87
40 15 85
25 45 231
6 39 645
23 34 129
40 35 493
1 46 244
11 4 528
14 24 606
18 13 777
10 20 314
8 37 325
28 4 17
12 25 89
43 27 631
4 48 31
32 36 42
7 16 977
1 18 989
15 13 258
42 49 41
11 22 37
5 23 150
13 5 223
12 18 817
26 40 571
19 20 519
8 31 539
36 33 265
30 44 573
8 14 429
27 46 391
23 31 255
49 8 432
9 39 720
23 9 586
22 4 480
19 47 132
24 26 690
21 49 818
7 47 312
21 6 437
24 32 615
41 29 280
21 32 451
11 24 946
50 8 699
5 34 703
37 38 716
11 40 310
21 22 684
12 5 409
15 8 461
13 43 297
11 20 76
3 2 193
50 5 357
10 26 349
39 28 205
9 29 192
40 24 705
36 7 545
46 26 799
13 29 967
23 36 498
40 43 512
39 2 546
49 37 83
30 3 631
35 9 993
32 37 59
17 33 375
22 44 656
35 46 886
16 3 718
9 46 273
5 24 415
4 41 916
38 16 718
38 7 794
30 19 821
35 50 997
22 47 265
14 46 106
25 26 681
7 48 875
28 6 231
44 6 433
44 48 52
19 12 915
22 17 838
38 19 966
10 13 719
26 38 516
26 43 433
20 35 102
7 43 128
42 27 295
48 20 933
32 40 933
49 9 89
22 33 64
21 34 681
29 42 927
30 47 452
40 36 413
13 33 705
39 50 200
28 2 452
13 11 420
25 3 739
12 10 228
43 32 752
38 8 430
27 47 222
34 6 760
14 21 828
23 47 392
24 2 751
14 4 207
29 33 250
24 37 212
16 4 861
46 44 223
4 37 257
13 16 708
48 10 165
38 50 508
11 29 572
30 14 573
24 29 640
20 1 362
29 34 560
3 43 895
15 2 926
43 36 62
30 28 958
12 50 595
28 27 197
43 23 645
13 44 149
41 20 493
2 21 741
34 8 440
6 27 146
11 31 217
10 40 861
11 21 264
24 6 642
31 9 291
43 49 869
24 33 723
45 34 531
23 46 631
46 38 505
9 8 999
36 47 179
25 33 864
2 47 400
43 25 484
24 41 119
33 3 989
28 17 332
17 11 319
9 12 829
2 23 172
9 16 700
45 23 127
27 3 480
29 15 289
29 28 309
2 16 917
43 35 194
1 31 552
20 15 551
45 44 996
43 14 494
5 47 929
11 16 606
41 27 431
28 14 921
25 38 426
49 31 835
45 42 762
39 45 621
4 13 145
15 38 156
26 19 183
14 2 470
22 30 670
11 7 196
14 47 49
9 48 282
21 17 992
16 50 113
43 30 36
11 9 971
1 35 95
11 38 859
10 25 585
12 34 664
13 38 982
46 36 108
39 18 105
37 1 970
47 25 845
42 17 883
42 26 243
45 41 172
17 5 165
8 20 757
48 13 668
31 38 504
3 7 312
15 48 842
26 39 203
21 43 857
10 3 822
48 2 472
27 8 640
48 24 874
15 34 723
28 12 157
5 45 690
45 9 750
23 16 301
29 50 651
7 41 896
15 18 843
22 25 207
31 15 138
1 39 285
8 11 268
50 22 471
22 7 351
19 40 540
48 46 846
41 19 941
30 39 311
44 23 952
33 45 303
41 44 642
8 17 887
18 25 778
19 11 749
18 11 262
32 3 339
17 37 866
47 48 987
6 3 591
32 12 332
37 44 823
23 35 377
10 18 752
35 42 958
24 30 926
20 12 888
32 48 452
33 31 827
18 26 119
14 42 970
26 45 793
3 37 681
33 23 533
13 40 186
41 37 903
50 24 804
8 10 478
27 24 952
17 45 269
1 34 196
44 19 812
50 21 134
25 23 673
4 34 323
27 21 187
47 45 562
16 35 310
31 34 746
16 43 33
1 11 10
8 42 616
32 42 814
43 48 770
13 21 347
42 38 124
9 22 642
8 7 177
31 10 831
24 39 606
21 38 23
35 27 286
32 46 26
5 15 75
14 25 701
41 21 773
22 6 9
45 49 734
42 31 835
16 19 346
42 13 996
3 46 502
17 29 846
9 2 799
31 37 143
25 20 294
19 3 326
39 19 577
18 37 880
39 15 783
14 23 850
49 7 880
50 27 569
46 15 368
8 47 30
45 2 224
30 26 629
32 26 773
45 29 455
4 2 582
17 15 368
22 49 52
17 2 757
32 50 390
39 46 457
19 34 19
43 8 502
42 43 404
28 21 195
28 37 601
6 19 798
12 4 669
25 2 35
25 31 790
19 9 622
35 7 37
7 32 656
22 8 365
24 22 119
24 23 467
4 33 23
21 4 556
49 36 290
10 4 549
20 32 118
9 7 282
12 48 527
7 21 379
49 11 537
28 45 415
4 20 725
3 23 339
44 7 168
46 34 482
9 50 265
7 6 961
7 10 370
29 47 754
30 13 776
14 33 581
24 17 974
48 3 99
50 36 151
14 48 469
26 28 555
44 32 462
27 38 479
29 8 267
50 26 33
50 6 66
12 39 972
26 34 210
47 44 842
33 26 770
45 6 923
38 43 117
15 49 708
8 46 591
46 43 134
31 39 859
48 33 271
32 34 113
16 46 33
32 35 158
3 42 223
28 9 628
18 2 441
27 22 358
2 27 749
17 23 794
17 48 261
38 33 393
28 25 191
48 40 467
35 24 275
16 8 67
32 27 343
47 46 254
30 42 505
20 47 87
20 46 330
46 33 987
13 24 672
39 49 211
21 5 223
27 49 932
8 18 362
1 42 269
40 39 69
27 31 244
16 22 971
13 26 644
49 35 450
41 10 785
42 47 769
20 14 730
15 33 920
23 48 949
6 38 443
10 14 979
22 12 407
41 14 574
45 37 784
23 40 453
15 22 23
35 28 968
5 30 411
48 27 865
15 37 555
21 44 558
29 32 899
2 33 979
50
13 24
24 32
40 11
21 5
27 39
49 21
12 15
36 14
28 17
10 20
24 10
30 49
30 32
30 35
8 21
20 14
16 43
16 13
25 2
23 39
10 5
43 1
35 39
12 48
43 44
6 14
34 23
37 7
38 22
48 7
1 36
37 31
49 15
7 3
43 39
49 7
50 16
11 1
5 47
24 17
4 32
6 35
50 38
45 14
15 34
15 8
49 13
44 15
28 38
1 38
