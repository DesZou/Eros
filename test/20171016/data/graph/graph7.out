466
467
457
575
618
482
506
373
522
515
442
419
468
482
342
420
303
349
315
585
617
359
583
417
385
442
558
494
468
414
423
467
491
390
491
464
383
392
486
437
464
511
502
481
426
385
474
417
445
442
